src/utils.ts

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs/promises'
+import zlib from 'zlib'
```

Comment:
Is `zlib` on all systems and shipped with all node distros?

Reply:
> Is `zlib` on all systems and shipped with all node distros?

Yep! So far I don't see any support issues in the NodeJS's zlib documentation.